Sources/PackageMetadata/PackageMetadata.swift

```diff
@@ -30,19 +30,60 @@ public struct Package {
         case sourceControl(url: URL)
     }
 
+    public struct Resource: Sendable {
+        public let name: String
+        public let type: String
+        public let checksum: String?
+        public let signing: Signing?
+    }
+
+    public struct Signing: Sendable {
+        public let signatureBase64Encoded: String
+        public let signatureFormat: String
+    }
```

Comment:
should we translate this into the SigningEntity? or is the client expect ted to do that?

Reply:
IDK, this just matches what is available today in `PackageVersionMetadata` when talking to the registry client.

Reply:
maybe a good enough start